android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -266,7 +264,7 @@ public void onCues(List<Cue> cues) {
             subtitleLayout.setCues(cues);
         }
 
-        // ExoPlayer.VideoListener implementation
+        // SimpleExoPlayer.VideoListener implementation
```

Comment:
```suggestion
        // ExoPlayer.VideoListener implementation
```

Reply:
Hmm.. I think we don't need comments anymore. because ComponentListener implement only one interface (It was required in v5)

If you don't want to remove it, please revert [a29d85d](https://github.com/react-native-video/react-native-video/pull/3337/commits/a29d85dd90f1a0d017b26ebe39694aea2b132ed1) commit